csharp/extractor/Semmle.Extraction.CSharp.DependencyFetching/NugetPackageRestorer.cs

```diff
@@ -604,6 +604,11 @@ private bool IsFeedReachable(string feed, int timeoutMilliSeconds, int tryCount,
                 {
                     httpClientHandler.ServerCertificateCustomValidationCallback = (message, cert, chain, _) =>
                     {
+                        if (chain is null || cert is null)
+                        {
+                            logger.LogWarning("Certificate validation trivially failed due to missing chain or certificate.");
```

Comment:
Two small points:

1. Should we give a bit more context here so it isn't so easily mistaken for a more generic error? How about "Dependabot proxy certificate validation [..]"?
2. Less important, but should we indicate which of the two is `null`?